.github/workflows/install.yml

```
@@ -104,10 +104,20 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
+
+      - name: Configure AWS Credentials
+        if: steps.gather.outcome == 'success'
```

actionlint (1.7.4)

property "gather" is not defined in object type {set-outputs: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)